### ISSUE TITLE
PyPI release with getchildren() fix

### ISSUE THREAD

**Issue by TryExceptElse** [ahmedelsayed-93/junit2html#7]
The [latest version of _vjunit_ on PyPI](https://pypi.org/project/vjunit/0.6.1/) still has the issue described in #5. A bugfix release with this fix included would be useful to allow easy installation and use with modern Python versions.

**Comment by RobertGawron** [comment]
+1; would be nice o have this fix in pypi.

**Comment by Penguin-zlh** [comment]
I find a way to resolve this question: update the file `vjunit.py`  that installed by pip

1. find `vjunit.py`
```
pip show vjunit
# use Location
cd /usr/local/lib/python3.10/dist-packages
cd vjunit
```
<img width="1920" height="442" alt="Image" src="https://github.com/user-attachments/assets/c27fffe1-2f7e-44d3-9f79-5aeead41238d" />

2. update `vjunit.py`, update `children = testcase.getchildren()` to `children = list(testcase)`
```diff
-               children = testcase.getchildren()
+               children = list(testcase)
```

3. after updated, vjunit can do well

<img width="987" height="69" alt="Image" src="https://github.com/user-attachments/assets/3a83a85b-90ff-4efd-b3b9-f8436755ac86" />

**Comment by TryExceptElse** [comment]
Yeah, this is the fix RomainC-lab implemented in 252be091. It's merged into master but a pypi release hasn't been made yet. I've been installing via git+https in the meanwhile, ex: `pip install git+https://github.com/ahsayde/vjunit@93f86cce5456a691e24c1558add48962342f8f3d`.

Thanks for taking the time to look at this though.
